src/Cache/FileCacheStorage.php

```diff
@@ -40,7 +40,7 @@ private function makeDir(string $directory): void
 	 */
 	public function load(string $key, string $variableKey)
 	{
-		return (function (string $key, string $variableKey) {
+		return (function () use ($key, $variableKey) {
 			[,, $filePath] = $this->getFilePaths($key);
```

Comment:
wdyt about moving this call before the function and capture it via closure-parameter, so we can declare the function `static`?

Reply:
Yes, please do that. Thanks.